GenderPayGap.WebUI/Controllers/Account/ChangePersonalDetailsController.cs

```diff
@@ -0,0 +1,80 @@
+﻿using GenderPayGap.Core.Interfaces;
+using GenderPayGap.Database;
+using GenderPayGap.WebUI.Classes;
+using GenderPayGap.WebUI.Helpers;
+using GenderPayGap.WebUI.Models.Account;
+using GovUkDesignSystem.Parsers;
+using Microsoft.AspNetCore.Mvc;
+
+namespace GenderPayGap.WebUI.Controllers.Account
+{
+    [Route("manage-account")]
+    public class ChangePersonalDetailsController : Controller
+    {
+        
+        private readonly IDataRepository dataRepository;
+
+        public ChangePersonalDetailsController(
+            IDataRepository dataRepository)
+        {
+            this.dataRepository = dataRepository;
+        }
+
+        // The user asks to change their personal details
+        // We direct them to a page that asks for their updated personal details
+        [HttpGet("change-personal-details")]
+        public IActionResult ChangePersonalDetailsGet()
+        {
+            var viewModel = new ChangePersonalDetailsViewModel();
+
+            // Get the current user
+            User currentUser = ControllerHelper.GetGpgUserFromAspNetUser(User, dataRepository);
+
+            // Fill the viewModel with the current user's information
+            viewModel.FirstName = currentUser.Firstname;
+            viewModel.LastName = currentUser.Lastname;
+            viewModel.JobTitle = currentUser.JobTitle;
+            viewModel.ContactPhoneNumber = currentUser.ContactPhoneNumber;
+            
+            // Return the Change Personal Details form
+            return View("ChangePersonalDetails", viewModel);
+        }
+        
+        // The user submits some new personal details
+        // We validate it (e.g. that all fields except contact phone number are filled in)
+        // Then we save the updates and return the user to the Manage Account page
+        [HttpPost("change-personal-details")]
+        [PreventDuplicatePost]
+        [ValidateAntiForgeryToken]
+        public IActionResult ChangePersonalDetailsPost(ChangePersonalDetailsViewModel viewModel)
+        {
+            // We don't validate contact number, since this is manually checked and we'll accept anything here

```

Comment:
The phone number doesn't get saved.
We still need to `ParseAndValidateParameters` the `ContactPhoneNumber`, we just don't need any validation attributes on the property on the view model